contrib/hadoop-store-builder/src/java/voldemort/store/readonly/swapper/HdfsFailedFetchLock.java

```diff
@@ -108,7 +110,12 @@
     public HdfsFailedFetchLock(VoldemortConfig config, Props props) throws Exception {
         super(config, props);
         fileSystem = HadoopUtils.getHadoopFileSystem(config, clusterDir);
-        initDirs();
+        try {
+            initDirs();
+        } catch(Exception ex) {
```

Comment:
Shall we catch the Throwable instead of Exception here? So that we could catch all types of error/exception.